geoips/plugins/modules/procflows/single_source.py

```diff
@@ -1379,7 +1379,7 @@ def call(fnames, command_line_args=None):
         reader_kwargs = {}
     compare_path = command_line_args["compare_path"]
     output_file_list_fname = command_line_args["output_file_list_fname"]
-    compare_outputs_module = command_line_args["compare_outputs_module"]
+    # compare_outputs_module = command_line_args["compare_outputs_module"]
```

Comment:
Ahh, so now it will find installed output checkers for indiviidual data types in any repo.  That is good.